test/bats.bats

```diff
@@ -37,7 +37,7 @@ fixtures bats
   run bats "$FIXTURE_ROOT/passing.bats"
   [ $status -eq 0 ]
   [ ${lines[0]} = "1..1" ]
```

Comment:
You want me to quote it too? I only quoted string compare with a space inside it. 

~~~
bats/test$ grep -RHn '[^"] = "[^" ]\+' 
bats.bats:33:  [ $output = "1..0" ]
bats.bats:39:  [ ${lines[0]} = "1..1" ]
suite.bats:9:  [ $output = "1..0" ]
suite.bats:15:  [ ${lines[0]} = "1..1" ]
suite.bats:32:  [ ${lines[0]} = "1..3" ]
suite.bats:41:  [ ${lines[0]} = "1..3" ]
suite.bats:48:  [ ${lines[0]} = "1..3" ]
~~~

but nothing with a space on the right side of the compare

~~~
bats/test$ grep -RHn '[^"] = "[^" ]\+ ' #  <= there's a space here.
~~~